pkg/base/replicated.go

```diff
@@ -152,7 +157,62 @@ func renderReplicated(u *upstreamtypes.Upstream, renderOptions *RenderOptions) (
 		}
 	}
 
-	return &commonBase, helmBases, nil
+	return &commonBase, helmBases, renderedKotsKinds, nil
+}
+
+func renderKotsKinds(upstreamFiles []upstreamtypes.UpstreamFile, renderedConfig *kotsv1beta1.Config, renderOptions *RenderOptions, builder *template.Builder) (map[string][]byte, error) {
+	fileContentsMap := make(map[string][]byte)
+
+	for _, upstreamFile := range upstreamFiles {
+		gvk := OverlySimpleGVK{}
+		if err := yaml.Unmarshal(upstreamFile.Content, &gvk); err != nil {
+			continue
+		}
+		gvkString := fmt.Sprintf("%s,Kind=%s", gvk.APIVersion, gvk.Kind)
```

Comment:
nit pick: can we just follow the `gvk.String()` pattern here and have a space like so:
```suggestion
		gvkString := fmt.Sprintf("%s, Kind=%s", gvk.APIVersion, gvk.Kind)
```